pylint/message/message_definition_store.py

```diff
@@ -9,6 +9,8 @@
 from collections.abc import Sequence, ValuesView
 from typing import TYPE_CHECKING
 
+import pytest
```

Comment:
You noticed this right? 😄

Reply:
Wow. I thought it was inside a test.

Reply:
I suggest doing this instead:

```python
            if message.may_be_emitted(sys.version_info):
                emittable.append(message)
            else:
                non_emittable.append(message)
```     

I think we need to make a follow-up MR to make things right (give py_version instead the whole way) and I think this is a looooot of effort just for the ``            "list-msgs-enabled",`` command.

Reply:
Can't we use the filter from `warnings` itself here? We use it in other places as well.

Reply:
How about keeping the deprecation warning until we fix it properly in a follow-up MR. (If we just filter the warning then when we make the parameter mandatory in 3.0 we're going to have to fix it anyway)

Reply:
Fine with me! This is only called for the CLI command right?

Reply:
Only in list-msg-enabled, yes. I don't think it's used that often.